File: java/SR-RBD/src/ddb/db/DbConnector.java
/**
 * 
 */
package ddb.db;

/** 
 * <!-- begin-UML-doc -->
 * <!-- end-UML-doc -->
 * @author User
 * @generated "UML to Java (com.ibm.xtools.transform.uml2.java5.internal.UML2JavaTransform)"
 */
public interface DbConnector {
	/** 
	 * <!-- begin-UML-doc -->
	 * <!-- end-UML-doc -->
	 * @param queryString
	 * @return
	 * @generated "UML to Java (com.ibm.xtools.transform.uml2.java5.internal.UML2JavaTransform)"
	 */
	public DatabaseTable query(String queryString) throws DBException;
    public void clearDatabase() throws DBException;
    public String dumpTable(String tableName) throws DumpTableException;
    public void importTable(String dump) throws ImportTableException;
}

File: trunk/C#/RBD/RBD/DbConnector.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace RBD
{
    interface DbConnector
    {
        DatabaseTable query(String queryString);
        void clearDatabase();
        String dumpTable(String tableName);
        void importTable(String dump);
    }
}
